plugins/module_utils/kibana.py

```diff
@@ -552,3 +552,42 @@ def get_agent_list(self):
         agent_no = agent_no + 1
       page_number = page_number + 1
     return agent_list_result
+
+  def get_fleet_server_hosts(self):
+    endpoint = 'fleet/settings'
+    result = self.send_api_request(endpoint, 'GET')
+    return result['item']['fleet_server_hosts']
+
+  def set_fleet_server_host(self, host):
+    endpoint = 'fleet/settings'
+    headers = {'kbn-xsrf': True}
+    body = {
+        'fleet_server_hosts': [
+          host
+        ]
+      }
+
+    body_json = dumps(body)
+
+    result = self.send_api_request(endpoint, 'PUT', headers=headers, data=body_json)
+    return result
+
+  def get_fleet_elasticsearch_hosts(self):
+    endpoint = 'fleet/outputs'
+    result = self.send_api_request(endpoint, 'GET')
+    for item in result['items']:
+      if item['id'] == "fleet-default-output" and item['type'] == 'elasticsearch':
+        return item['hosts']
+
+  def set_fleet_elasticsearch_host(self, host):
```

Comment:
may be a good idea to take the `host` entry as a list to support an eventual world where there are multiple targets (which will hopefully be later this year)